### PR TITLE
Implemented a disk based store.

### PR DESCRIPTION
**Pull request by stuartskelton**
The why... is to save the chunks on the disk so that when you are working offline on a project you can still open the chunked logs.

### CODE REVIEW

**Comment by stuartskelton** [comment]
booo..., I have bumped the version of Dist::Zilla to 5.047
As of Dist::Zilla 6.000  https://github.com/rjbs/Dist-Zilla/blob/master/Changes
- Dist::Zilla now requires perl v5.14.0.  It will still happily build
          dists that run on whatever version of perl you like.

**Comment by stuartskelton** [comment]
Well that didn't work, it looks like something more robust is in need

SEE: https://github.com/SineSwiper/Dist-Zilla-TravisCI/blob/master/.travis.yml

File: lib/Log/Log4perl/Appender/Chunk/Store/File.pm [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    unless ( -d $logging_dir ) {
+        eval { File::Path::mkpath($logging_dir) };
+        if ($@) {
+            $LOGGER->trace("Couldn't create $logging_dir: $@");
+            return;
+        }
+    }
+    $LOGGER->trace("Storing chunk ".$chunk_id);
+    open my $FH, '>', File::Spec->catfile($logging_dir, $chunk_id);
+    print $FH $big_message;
```
Comment (jeteve) [inline lib/Log/Log4perl/Appender/Chunk/Store/File.pm]: use File::Slurp (with utf8)

File: lib/Log/Log4perl/Appender/Chunk/Store/File.pm [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    close $FH;
+    return 1;
```
Comment (jeteve) [inline lib/Log/Log4perl/Appender/Chunk/Store/File.pm]: log 'stored chunk ID *chunk ID* in *this place*

**Comment by jeteve** [comment]
Nice one!

**Pull request merged**